client/coral-embed-stream/src/Embed.js

```diff
@@ -158,6 +159,11 @@ class Embed extends Component {
               clearNotification={this.props.clearNotification}
               notification={{text: null}}
             />
+          <LoadMore
+            asset_id={asset.id}
+            comments={asset.comments}
+            moreComments={asset.commentCount > asset.comments.length}
```

Comment:
Isn't the `commentCount` the total comment count? This will be always greater than the length of the comments array if the limit is 5.